packages/authgear-react-native/ios/AGAuthgearReactNative.h

```diff
@@ -5,7 +5,7 @@
 #import <React/RCTBridgeModule.h>
 #import <React/RCTEventEmitter.h>
 
-@interface SGSkygearReactNative : RCTEventEmitter <RCTBridgeModule>
+@interface AGAuthgearReactNative : NSObject <RCTBridgeModule>
```

Comment:
Why do we change the parent class?

Reply:
Inheriting `RCTEventEmitter` needs to implements `supportedEvents`, which is no longer needed and removed in a previous commit. So just revert to use a plain `NSObject`.